mock/datasets/no2.data.mdx

```diff
@@ -50,6 +50,9 @@ layers:
   - id: no2-monthly
     stacCol: no2-monthly
     name: No2 PT
+    media:
+      src: ::file ./img-placeholder-3.jpg
+      alt: Placeholder Image
```

Comment:
This is how a media can be added to a layer @faustoperez  I am just reusing an image that was already in the folder (`img-placeholder-3.jpg`) but let's go over how we can add an actual file later together 👯